pcsx2/ps2/Iop/PsxBios.cpp

```diff
@@ -58,7 +58,7 @@ void psxBiosReset()
     flush_stdout(true);
 }
 
-// Called for Playstation BIOS calls at 0xA0, 0xB0 and 0xC0 in kernel reserved memory (seemingly by actually calling those addresses)
+// Called for PlayStation BIOS calls at 0xA0, 0xB0 and 0xC0 in kernel reserved memory (seemingly by actually calling those addresses)
```

Comment:
Is there really a need to update code comments?

Reply:
Why not? I'm not bothered.